### PR TITLE
Added Find duplicate elements from an array

### PR DESCRIPTION
**Pull request by RaiyanMahin**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

YES.

### Description

Given an array of N elements , find all duplicate elements from it. In Competitive Programming, some times we need to do this to solve our problem.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Fixed Issue #3517

### CODE REVIEW

File: C-Plus-Plus/other/find_duplicate_elements_from_an_array.cpp [hunk 1]
```diff
@@ -0,0 +1,59 @@
+/*
+Given an array of N elements, each element's occurrence in the array can be more than once. Find all elements whose occurrence is more than 1.
+Formally find all duplicate elements of the array. This can be done with the help of map and set data structure.
+*/
+
+#include <bits/stdc++.h>
+using namespace std;
+
+int ar[100009];
```
Comment (atarax665) [inline C-Plus-Plus/other/find_duplicate_elements_from_an_array.cpp]: You can declare the array locally after taking input N from the user.

File: C-Plus-Plus/other/find_duplicate_elements_from_an_array.cpp [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        cout << i << ' ';
+    cout << endl;
+    return;
+}
+
+int main()
+{
+    cout << "Input array size\n";
+    int N; 
+    cin >> N;
```
Comment (atarax665) [inline C-Plus-Plus/other/find_duplicate_elements_from_an_array.cpp:37]: Add proper statements to prompt user to input data.

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Comment by RaiyanMahin** [comment]
@atarax665  I made all the changes. Kindly Review.

**Comment by RaiyanMahin** [comment]
@iamrajiv @ankitaggarwal23 kindly review

**Comment by RaiyanMahin** [comment]
@bhav09 @atarax665 kindly review

**Review by atarax665 — APPROVED** [review]
LGTM

**Comment by RaiyanMahin** [comment]
@atarax665  Thanks. Any other's review is required?
Or you can merge it? 😁

**Review by iamrajiv — APPROVED** [review]

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**